x/mongo/driver/topology/connection.go

```diff
@@ -101,7 +102,7 @@ func (c *connection) processInitializationError(err error) {
 
 // connect handles the I/O for a connection. It will dial, configure TLS, and perform
 // initialization handshakes.
-func (c *connection) connect(ctx context.Context) {
+func (c *connection) connect(ctx context.Context, monitor *event.PoolMonitor) {
```

Comment:
Instead of passing it in directly, `connection` should have an option to set a pool monitor. This makes it consistent with the `Server` type.

Reply:
done